vulnerabilities/models.py

```diff
@@ -2927,17 +2929,19 @@ class ImpactedPackage(models.Model):
 
     base_purl = models.CharField(
         max_length=500,
-        blank=True,
+        blank=False,
         help_text="Version less PURL related to impacted range.",
     )
 
     affecting_vers = models.TextField(
         blank=True,
+        null=True,
```

Comment:
IMO, we should have a check that at least one of `affecting_vers` or `fixed_vers` should exist while creating/saving an object